packages/contracts/test/invariant/diamond/facets/PoolFacetHandler.sol

```diff
@@ -0,0 +1,266 @@
+        MockChainLinkFeed _stableUsdPriceFeed,
+        MockChainLinkFeed _ethUsdPriceFeed,
+        UbiquityPoolFacet _ubiquityPoolFacet,
+        address _admin,
+        address _user,
+        MockCurveStableSwapNG _curveDollarPlainPool
+    ) {
+        collateralTokenPriceFeed = _collateralTokenPriceFeed;
+        stableUsdPriceFeed = _stableUsdPriceFeed;
+        ethUsdPriceFeed = _ethUsdPriceFeed;
+        ubiquityPoolFacet = _ubiquityPoolFacet;
+        admin = _admin;
+        user = _user;
+        curveDollarPlainPool = _curveDollarPlainPool;
+    }
+
+    /**
+     * @notice Manipulates the Ubiquity Dollar price to a value above a set threshold.
+     * @dev This function assumes the new dollar price is within the specified range (greater than 1e18 and less than 2e18).
+     * It then updates the mocked Curve pool parameters and adjusts the collateral ratio in the UbiquityPoolFacet.
+     * @param newDollarPrice The new price for Ubiquity Dollar, expected to be within the range of 1e18 to 2e18.
+     */
+    function setDollarPriceAboveThreshold(uint256 newDollarPrice) public {
+        vm.assume(newDollarPrice > 1e18 && newDollarPrice < 2e18);
+
+        vm.prank(admin);
+        curveDollarPlainPool.updateMockParams(newDollarPrice);
+
+        uint256 reductionFactor = newDollarPrice.sub(1e18).div(1e16);
+        uint256 newCollateralRatio = uint256(1e6).sub(reductionFactor);
+
+        ubiquityPoolFacet.setCollateralRatio(newCollateralRatio);
+    }
+
+    /**
+     * @notice Manipulates the Ubiquity Dollar price to a value below a set threshold.
+     * @dev This function assumes the new dollar price is within the specified range (greater than or equal to 0.5e18 and less than 1e18).
+     * It then updates the mocked Curve pool parameters and adjusts the collateral ratio in the UbiquityPoolFacet accordingly.
+     * @param newDollarPrice The new price for Ubiquity Dollar, expected to be within the range of 0.5e18 to 1e18.
+     */
+    function setDollarPriceBelowThreshold(uint256 newDollarPrice) public {
+        vm.assume(newDollarPrice >= 0.5e18 && newDollarPrice < 1e18);
+
+        vm.prank(admin);
+        curveDollarPlainPool.updateMockParams(newDollarPrice);
+
+        uint256 increaseFactor = uint256(1e18).sub(newDollarPrice).div(1e16);
+        uint256 newCollateralRatio = uint256(1e6).add(increaseFactor);
+
+        ubiquityPoolFacet.setCollateralRatio(newCollateralRatio);
+    }
+
+    /**
+     * @notice Manipulates the redemption delay in blocks for UbiquityPoolFacet.
+     * @dev This function allows the admin to set a delay in blocks before a redemption can be completed.
+     * It assumes the caller is the admin and pranks the transaction as the admin to set the redemption delay.
+     * @param delay The number of blocks to set as the redemption delay.
+     */
+    function setRedemptionDelay(uint256 delay) public {
+        vm.prank(admin);
+        ubiquityPoolFacet.setRedemptionDelayBlocks(delay);
```

Comment:
This handler method adds too much unnecessary reverts on collection redemption. We'd better:
1. Remove `setRedemptionDelay()` handler method
2. Set `redemptionDelayBlocks` to 0 (in order not to wait for passed blocks for collecting redemption) in [UbiquityPoolFacet.invariant.t.sol](https://github.com/ubiquity/ubiquity-dollar/blob/dfff4f314386ef0ec4d0fb6091367623823480e0/packages/contracts/test/invariant/diamond/facets/UbiquityPoolFacet.invariant.t.sol)'s `setUp()`

This way we more or less simulate real user flow.

Reply:
The setRedemptionDelay() method has been removed